samples/net/sockets/echo_server/boards/nucleo_g431rb.conf

```diff
@@ -0,0 +1,52 @@
+# Generic networking options
```

Comment:
This sample board config should be in a separate commit.

Please only place here board specific stuff so that the main prj.conf in that sample will be the master information and here we just tweak things needed for this board. Do not place config options specific to your local setup here (like IP addresses etc).

Reply:
The `prj.conf` file needs changes as shown in `nucleo_g431rb.conf`, otherwise the sample will not build (excess of on-board RAM).

IMHO, it would be better to have a complete setup there for this particular development board, so other users can reproduce this setup easily.

Reply:
Sorry, but this is not a way here. The prj.conf will be the master of the config and the config in board/ directory can then add or adjust the configuration parameters. This way if any new config options are added in prj.conf, the compilation for this board will automatically use them and we do not need to adjust two files.